src/alignment/PSSMCalculator.cpp

```diff
@@ -171,7 +168,6 @@ void PSSMCalculator::computeNeff_M(float *frequency, float *seqWeight, float *Ne
             }
         }
         Neff_M[pos] = (w_M < 0) ? 1.0 : Nlim - (Nlim - 1.0) * MathUtil::fpow2(scale * w_M);
-//        fprintf(stderr,"M  i=%3i  ncol=---  Neff_M=%5.2f  Nlim=%5.2f  w_M=%5.3f  Neff_M=%5.2f\n",pos,Neff_HMM,Nlim,w_M,Neff_M[pos]);
```

Comment:
Please leave comments in the source code. They are used to find bugs.

Reply:
I have removed the two commits which deleted debug prints.